requirements.txt

```diff
@@ -1,28 +1,40 @@
 appdirs==1.4.2
+autopep8==1.3.1
 awscli==1.10.5
 boto3==1.4.4
 botocore==1.5.19
 colorama==0.3.3
 coverage==4.3.4
 decorator==4.0.11
 docutils==0.13.1
+extras==1.0.0
+fixtures==3.0.0
 futures==3.0.5
 hurry.filesize==0.9
 Jinja2==2.9.5
 jmespath==0.9.1
 jsonpath-rw==1.4.0
+linecache2==1.0.0
 MarkupSafe==0.23
 moto==0.4.31
+mox3==0.21.0
 netaddr==0.7.18
 nose==1.3.7
 numpy==1.12.0
 packaging==16.8
+pbr==2.0.0
 ply==3.10
 psutil==5.1.3
 pyasn1==0.2.3
+pycodestyle==2.3.1
+pyfakefs==3.1
 pyparsing==2.1.10
 python-dateutil==2.6.0
+python-mimeparse==1.6.0
 rsa==3.3
 s3transfer==0.1.10
 six==1.10.0
+testtools==2.2.0
+traceback2==1.4.0
+unittest2==1.1.0
 virtualenv==15.1.0
```

Comment:
Have you included all your dev reqs here? It looks like a lot of these aren't actual code dependencies for SA